ParkingZoneApp/Areas/Admin/Views/ParkingSlot/Index.cshtml

```diff
@@ -4,46 +4,56 @@
     Layout = "~/Views/Shared/_Layout.cshtml";
 }
 
-<h1>Parking slots for @Html.DisplayNameFor(model => model)</h1>
+<h1>Slots for Parking Zone</h1>
 
 <p>
-    <a asp-action="Create">Create New</a>
+    <a asp-action="Create" asp-controller="ParkingSlot" asp-route-parkingZoneId="@ViewData["parkingZoneId"]" class="btn btn-success">Create New</a>
 </p>
 <table class="table">
     <thead>
         <tr>
             <th>
-                @Html.Display("Number")
+                <h6>No</h6>
             </th>
             <th>
-                @Html.Display("Category")
+                <h6>Number</h6>
```

Comment:
this way is called "dehqoncha", lets make it in a MVC way with HTML.displayfor() or other